integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/End2EndTestITStep.java

```diff
@@ -0,0 +1,257 @@
+package com.redhat.service.bridge.integration.tests.steps;
+
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.hamcrest.Matchers;
+import org.testcontainers.shaded.org.awaitility.Awaitility;
```

Comment:
IMHO this class should be imported from `org.awaitility`, it doesn't have sense to use testcontainers dependency for this.

Reply:
That will allow you to drop `test-utils` dependency.
Less dependencies = avoiding dependency hell.